data/main.ui

```diff
@@ -40,7 +40,7 @@
                       <object class="GtkButton" id="reset-button">
                         <property name="valign">start</property>
                         <property name="visible">True</property>
-                        <property name="label">Reset</property>
+                        <property name="label" translatable="yes" context="notebook container">Reset</property>
```

Comment:
I had been seeing that on other cases, i18n-able labels on UI files, the `context` attribute was defined so it could be used from the `po` side. As it doesn't seem mandatory, we could remove it though.